labs/b1.md

```diff
@@ -101,12 +95,12 @@ verify your conjectures.
 
 3. What is the root directory in Linux filesystems?
```

Comment:
Emphasize that this should be answered conceptually (like what exactly is the root directory)? instead of just sticking a `/` in the answer box